modules/RTC/JitsiLocalTrack.js

```diff
@@ -395,6 +395,9 @@ export default class JitsiLocalTrack extends JitsiTrack {
                         this._unregisterHandlers();
                         this.stopStream();
                         this._setStream(null);
+
+                        // Send the videoType message to the bridge.
+                        this.isVideoTrack() && this.conference.sendBridgeVideoTypeMessage(this);
```

Comment:
`sendBridgeVideoTypeMessage` should not be a public method on `JitsiConference`. Can this be done at the end of `JitsiConference._removeLocalTrackAsMute`?

Reply:
No it cannot be done at the end of JitsiConference._removeLocalTrackAsMute since the jitsiLocalTrack.isMuted() doesn't return true until after the `this.stream` on the jitsiLocalTrack is set to `null` which happens here https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/RTC/JitsiLocalTrack.js#L397.
Also I see that the track mute status in presence is also being set [here](https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/RTC/JitsiLocalTrack.js#L343) after `jitsiLocalTrack.mute()` is executed. How is this any different ?

Reply:
Can you please rename `sendBridgeVideoTypeMessage` to be `_sendBridgeVideoTypeMessage`? It's not a public method for lib-jitsi-meet library users to call.